contracts/SSVNetwork.sol

```diff
@@ -104,13 +150,56 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         emit OperatorRemoved(operatorId);
     }
 
-    function updateOperatorFee(uint64 operatorId, uint64 fee) external {
+    function declareOperatorFee(uint64 operatorId, uint64 fee) onlyOperatorOwnerOrContractOwner(operatorId) external {
```

Comment:
In general with uint64 highest fee possible is ~18 .  We need to add shrink and expand functions (affects many other places as well)